jwql/website/jwql_proj/settings.py

```diff
@@ -52,6 +56,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'revproxy.apps.RevProxyConfig'
```

Comment:
@york-stsci `revproxy` is giving me some issues locally when I try to run jwql/import methods from the website modules. It's originating from the django `setup()` call.

We should add `revproxy` to our dependency list, but even after adding revproxy I am getting a ModuleNotFoundError:
`ModuleNotFoundError: No module named 'revproxy.apps'`

Reply:
Oops. That's a way that I was trying to get the preview to load, and it's not being used at all anymore, but I forgot to delete it. I'm going to do my best to eliminate it entirely in my next update to this PR, but please check to make sure, because my own environment isn't seeing it because I actually installed revproxy.

Reply:
Thank you!